main.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'json'
+require 'securerandom'
+
+MEMOS_FILE_PATH = 'memos.json'
+
+class Memo
+  include Rack::Utils
+  attr_writer :title, :text
+
+  def initialize(title, text)
+    @title = title
+    @text  = text
+  end
+
+  def to_json(*args)
+    { 'title' => @title, 'text' => @text }.to_json(*args)
```

Comment:
```suggestion
  def to_json(...)
    { 'title' => @title, 'text' => @text }.to_json(...)
```

という書き方もあります

https://docs.ruby-lang.org/ja/3.4/doc/symref.html#period

あと、シンボルを使っても結果は同じになるかも知れません。（好みの問題ですが）

```suggestion
  def to_json(...)
    { title: @title, text: @text }.to_json(...)
```